.github/pull_request_template.md

```diff
@@ -1,22 +1,32 @@
-<!--
-  Copy the task id from the bottom of the sidebar and paste it after FTRACK-
+<!--(
+  Copy the id and paste it to the appropriate CLICKUP-<id> / FTRACK-<id> /  SENTRY-<id> / ZENDESK-<id> link.
+  Please remember to remove the unused ones.
 -->
+Resolves : 
 
-Resolves FTRACK-
+* CLICKUP-
+* FTRACK-
+* SENTRY-
+* ZENDESK-
 
-- [ ] I have added automatic tests where applicable
-- [ ] The PR title is suitable as a release note
-- [ ] The PR contains a description of what has been changed
-- [ ] The description contains manual test instructions
+- [ ] I have added automatic tests where applicable.
+- [ ] The PR contains a description of what has been changed.
+- [ ] The description contains manual test instructions.
+- [ ] The PR contains update to the release notes.
+- [ ] The PR contains update to the documentation.
 
-## Changes
+<!--
+  What platform / browser this PR has been tested with.
+-->
 
+This PR has been tested on :
+- [ ]
```

Comment:
I don't think this is generic enough to be applicable as a default PR template description. Any repositories that wants to track which platsforms and browsers it has been tested on should add their own PR template imo.

Reply:
thought the idea was to have templates to further customise based on specific needs (hence keeping this open wide) but sure.